output.tf

```diff
@@ -0,0 +1,3 @@
+output "aks" {
+  value = azurerm_kubernetes_cluster.aks
+}
```

Comment:
I imagine we'll need the kubeconfig as an output for when this gets plugged into the deployer.

Reply:
Oh yeah, this is more of just a place holder ;)